docs/dev/new_check_howto.md

```diff
@@ -440,19 +470,33 @@ For Agent versions >= 6.12:
 "C:\Program Files\Datadog\Datadog Agent\bin\agent.exe" integration install -w /path/to/wheel.whl
 ```
 
-[1]: https://virtualenv.pypa.io/en/stable
-[2]: https://github.com/DataDog/integrations-core/blob/master/docs/dev/python.md
-[3]: https://github.com/DataDog/integrations-extras
-[4]: https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev
-[5]: https://docs.datadoghq.com/developers/metrics/agent_metrics_submission/
-[6]: https://github.com/DataDog/datadog-agent/blob/6.2.x/docs/dev/checks/python/check_api.md
-[7]: https://docs.pytest.org/en/latest
-[8]: https://tox.readthedocs.io/en/latest
-[9]: https://datadoghq.dev/integrations-core/setup/#ddev
-[10]: https://docs.datadoghq.com/developers/integrations/check_references/#configuration-file
-[11]: https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file
-[12]: https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file
-[13]: https://docs.datadoghq.com/developers/integrations/check_references/#service-check-file
-[14]: https://packaging.python.org/en/latest/tutorials/packaging-projects/
-[15]: https://docs.datadoghq.com/agent/
-[16]: https://datadoghq.dev/integrations-core/meta/config-specs/
+## Integration publishing checklist
+
+After you complete your integration, refer back to this list to make sure you've got everything you need.
+
+To consider an Agent-based integration complete, and thus ready to be included in the core repository and bundled with the Agent package, several prerequisites must be met:
```

Comment:
I'm not sure to understand this. Integrations in -extras are not bundled with the agent and not included in the integrations-core repository. Customers will need to install the integration manually and they are not officially maintained by Datadog.

Reply:
Good call, removing this paragraph